docs/protocol/extension_statistics.md

```diff
@@ -114,6 +115,12 @@ $model_stat =
   model/version. So, for example, "inference_stats":"success"
   indicates the number of successful inference requests for the model.
 
+- "response_stats" : The aggregate decoupled response statistics for the
+  model/version. For example, { "key" : { "response_stats" : "success" } }
+  indicates the aggregate statistics of successful decoupled response at "key"
+  for the model, where "key" identifies between different decoupled responses
+  generated by the model.
```

Comment:
> The aggregate decoupled response statistics for the model/version.

I think we need to clarify `model/version`. Is it the specific version of a specific model?

>  indicates the aggregate statistics of successful decoupled response at "key" for the model

I am a little bit confused what `at "key" for the model` should mean.

>   where "key" identifies between different decoupled responses generated by the model.

` identifies between different decoupled responses` <- not sure what you mean by this.

Reply:
> I think we need to clarify model/version. Is it the specific version of a specific model?

It is a specific version of a specific model, because the backend views different versions of a model different models. Let's use "model" for it, because except one or two other places, the rest of the docs refers to "model".

> I am a little bit confused what at "key" for the model should mean.

It identifies between different decoupled responses generated by the model. For example, a model generates 3 responses, the key can be "0", "1" and "2". 

> identifies between different decoupled responses <- not sure what you mean by this.

Basically a response, even an empty one, should be mapped to a uniquely identifiable value -> "key". It is ultimately the backend responsibility for supplying valid keys for grouping response statistics. i.e. send the response statistics at the right order to ensure it is mapped to the correct index ("0", "1", "2", ...).

I added a comment below to ask the user to check with the backend on the set of keys and what each means.